docs/admin/maintenance/noble_migration.rst

```diff
@@ -47,16 +47,19 @@ take place shortly after your selected automated restart time, which you can adj
 
 If you have any questions, please reach out to Support.
 
-Semi-automated upgrade
+Semiautomated upgrade
 ----------------------
 
-* Ensure your Admin Workstation has been upgraded to SecureDrop 2.12
-* Open a Terminal
-* Run ``cd Persistent/securedrop``
-* Run ``./securedrop-admin backup`` to :doc:`take a backup <backup_and_restore>`
-* Run ``./securedrop-admin noble_migration``
+We recommend setting aside at least an hour for this process.
+
+* Log into your *Admin Workstation* and open a terminal window.
+* Run ``cd Persistent/securedrop``.
```

Comment:
Something that didn't occur to me in the previous review - should this path be `~/Persistent/securedrop`? This will almost always work either way, but there's a chance the user could have opened a Terminal and navigated away from home, at which point that wouldn't get them to the right place.

Reply:
I think if we just asked them to open a terminal, it's reasonable to assume that they didn't go somewhere else first.

I think we can cater to the more common case that they're doing exactly what we asked, which means slightly less typing for everyone else.